conf/application.conf

```diff
@@ -72,13 +72,10 @@ play.modules.enabled += "config.Module"
 # ~~~~
 # Set to the object handler for migrated 2.3 services
 
-Csrf-Bypass-value = "bm9jaGVjaw=="
-
 play.i18n.langs = ["en", "cy"]
 
 play.filters.csrf.header.bypassHeaders {
   X-Requested-With = "*"
-  Csrf-Bypass = "nocheck"
 }
```

Comment:
I think we can get rid of this entirely - as we don't need any bypass configuration